pages/stories/[id].js

```diff
@@ -0,0 +1,21 @@
+import { useRouter } from "next/router";
+import StoryDetailCard from "@/components/StoryDetailCard";
+import stories from "@/public/db";
+
+function StoryDetailPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  console.log("stories:", stories);
+  console.log("id:", id);
+
+  const story = stories.find((story) => story.id === parseInt(id));
+
+  if (!story) {
+    return <div>Story not found</div>;
```

Comment:
Maybe a `p` instead of a `div`? 👀

Reply:
great idea as always! 
thx for your time and knowledge!
changes done, please check the new PR